### PR TITLE
AREDN exporter fixes

### PR DESCRIPTION
**Pull request by matfra**
I've added a bare minimal "node_uname_info" which brings us out of the box compatibility with Node Exporter Full: the most popular Grafana dashboard. https://grafana.com/grafana/dashboards/

While adding an HELP and TYPE block to the "node_details_*" metrics, I thought their name could be improved. Since these metrics are specific to AREDN I propose:
- node_aredn_info
- node_aredn_meshrf

Additionally, this PR contains 2 small fixes for a mistyped metric and a (what I believe is) a redundant gauge.

Test:
- I haven't tested this yet. I need to figure out how build the package first.

Notes:
- The indentation issues showing on the diff are due to a mix of whitespaces and tabs. I'm happy to fix this in a dedicated no-op commit later.

### CODE REVIEW

File: utils/prometheus-exporter/files/metrics/lqm.lua [hunk 1]
```diff
@@ -47,7 +47,6 @@ if f then
         "distance",
         "exposed",
         "hidden",
-        "last_tx",
```
Comment (aanon4) [inline utils/prometheus-exporter/files/metrics/lqm.lua]: Why did you remove this one? Both last_tx and last_tx_total are metrics and are different.

**Review by aanon4 — COMMENTED** [review]
Just one question.

File: utils/prometheus-exporter/files/metrics/lqm.lua (hunk 1 above)
Comment (matfra) [reply utils/prometheus-exporter/files/metrics/lqm.lua]: My mistake, I thought it was redundant with last_tx total.

**Review by aanon4 — APPROVED** [review]
Thanks for this!
